Packages/com.unity.inputsystem/Documentation~/HID.md

```diff
@@ -1,49 +1,51 @@
 # HID Support
 
-Human Interface Device (HID) is a [specification](https://www.usb.org/hid) to describe peripheral user input devices connected to computers via USB or Bluetooth. HID is commonly used to implement devices such as gamepads, joysticks, or racing wheels.
+A Human Interface Device (HID) is a [specification](https://www.usb.org/hid) to describe peripheral user input devices connected to computers via USB or Bluetooth. HIDs are commonly used to implement devices such as gamepads, joysticks, or racing wheels.
```

Comment:
I think HID should be singular, because it's a single specification, not several different ones. We should revert the edit on the second sentence. (See https://en.wikipedia.org/wiki/USB_human_interface_device_class)